roofit/roofitcore/inc/RooAbsCategory.h

```diff
@@ -49,7 +52,7 @@ class RooAbsCategory : public RooAbsArg {
   // Value accessors
   virtual value_type getCurrentIndex() const ;
   /// Retrieve a batch of category values for events in the range [begin, begin+batchSize).
-  virtual RooSpan<const value_type> getValBatch(std::size_t /*begin*/, std::size_t /*batchSize*/) const {
+  virtual RooSpan<const value_type> getValues(BatchHelpers::RunContext&, const RooArgSet* = nullptr) const {
```

Comment:
Try to remove this. It think it's not needed. When batch values for categories are desired, it can easily be added.